packages/components/src/components/contract-card/contract-card-items/lookbacks-card-body.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { getDisplayStatus, getCardLabels, isCryptocurrency, CONTRACT_TYPES } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import ContractCardItem from './contract-card-item';
+import MobileWrapper from '../../mobile-wrapper';
+import Money from '../../money';
+import Text from '../../text';
+import ArrowIndicator from '../../arrow-indicator';
+import { ResultStatusIcon } from '../result-overlay/result-overlay';
+import { TGeneralContractCardBodyProps } from './contract-update-form';
+
+type TLookBacksCardBody = Pick<TGeneralContractCardBodyProps, 'contract_info' | 'currency' | 'is_sold'> & {
+    progress_slider_mobile_el: React.ReactNode;
+    indicative?: number | null;
+};
+
+const LookBacksCardBody = ({
+    contract_info,
+    currency,
+    is_sold,
+    indicative,
+    progress_slider_mobile_el,
+}: TLookBacksCardBody) => {
+    const { buy_price, contract_type, sell_price, profit, multiplier } = contract_info;
+    const { INDICATIVE_PRICE, MULTIPLIER, PROFIT_LOSS, POTENTIAL_PROFIT_LOSS, PAYOUT, PURCHASE_PRICE } =
+        getCardLabels();
+
+    const getPayoutLimit = (contract_type?: string, multiplier?: number) => {
+        let base: string;
+        switch (contract_type) {
+            case CONTRACT_TYPES.LB_PUT:
+                base = 'High - Close';
+                break;
+            case CONTRACT_TYPES.LB_CALL:
+                base = 'Close - Low';
+                break;
+            case CONTRACT_TYPES.LB_HIGH_LOW:
+                base = 'High - Low';
+                break;
+            default:
+                base = '';
+        }
```

Comment:
should the base be localized? if yes, it's better to localize it here:

```suggestion
        let base: string | React.ReactNode = '';
        if (contract_type === CONTRACT_TYPES.LB_PUT) {
            base = <Localize i18n_default_text='High - Close' />;
        } else if (contract_type === CONTRACT_TYPES.LB_CALL) {
           base = <Localize i18n_default_text='Close - Low' />;
        }  else if (contract_type === CONTRACT_TYPES.LB_HIGH_LOW) {
           base = <Localize i18n_default_text='High - Low' />;
        }

```
I suggest considering using `if else` because it does the same in 8 lines instead of 14 :) what do you think?

Reply:
Good catch, will apply, thank you